android/src/app/organicmaps/widget/placepage/PlacePageWikipediaFragment.java

```diff
@@ -81,6 +85,13 @@ private Spanned getShortDescription()
   private void updateViews()
   {
     mPlaceDescriptionView.setText(getShortDescription());
```

Comment:
If the view is hidden, why all other setters/methods are still called?

Reply:
Good point, I moved them inside the if.  See new version.